.github/workflows/lint.yml

```diff
@@ -5,6 +5,9 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: ShellCheck
-        uses: ludeeus/action-shellcheck@1.1.0
+      - uses: actions/checkout@v3
+      - name: Run ShellCheck
+        uses: ludeeus/action-shellcheck@master
```

Comment:
While this matches with their usage examples, it now adds the risk of things breaking especially if the action will have a major release (e.g. from v2 to v3). 

Looking at it now, v2 was released around 2 months ago, while we used v1. This v2 doesn't bring any breaking changes fortunately, but other major release may bring some.

Another reason why `master` tag looks risky is that changes may be added that weren't tested or are unstable, leading to the small chance of breaking the action. 


Therefore, my question is: should we better use a version tag (in this case `2.0.0`) rather than the `master` tag?

Reply:
That sounds like a good idea, but unfortunately it will not work for this specific action.

According to [Using tags for release management for GitHub Actions](https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-tags-for-release-management):
> Move the major version tag (such as v1, v2) to point to the Git ref of the current release.

Looking at the tags for that action, they don't have `v2` or `2`, they only have `2.0.0`. Therefore, `@2` will reference to nothing 😞